### PR TITLE
Set up a 'Hello Microverse' project

### PR DESCRIPTION
**Pull request by Frelly0**
Add a index.html.
add a style.css.
Add a README.md.
Add lenters for html,css.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,233 @@
...
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```
Comment (V-Blaze) [inline README.md:173]: - [OPTIONAL] Just as I pointed out above, I'll suggest you remove this, it is a placeholder text that explains to you what to do here, You are meant to remove this while customizing your readme :+1: :smile:

File: README.md [hunk 2]
```diff
@@ -1 +1,233 @@
...
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
```
Comment (V-Blaze) [inline README.md:205]: - [OPTIONAL] Just as I pointed out above, I'll suggest you remove this, it is a placeholder text that explains to you what to do here, You are meant to remove this while customizing your readme :+1: :smile:

File: README.md [hunk 3]
```diff
@@ -1 +1,233 @@
...
+
+
+
+<!-- LICENSE -->
+
+<!-- ## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (V-Blaze) [inline README.md:231]: - [ ] You have done great so far :+1: But I think there are still some improvements you need to make on your readme file. I can see you commented out the MIT license section, According to Microverse, the only optional section is the FAQ, the rest of the section needs to be present, I'll suggest you uncomment only lines `227 to 229`, then create a new file in your project with the name `LICENSE`.  you can [use this ](https://github.com/V-Blaze/gvash-planes/blob/dev/LICENSE)and then customize it for your project. :+1: :smile:

File: README.md [hunk 4]
```diff
@@ -1 +1,233 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
```
Comment (V-Blaze) [inline README.md:208]: - [OPTIONAL] For this section you can add `If you like this project, you can support it by giving it a start `. :smile:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @Frelly0  , 

##  You submitted this project as a re-review, but there is no previous review on this pull request or on this branch, If it happens that you deleted the branch or the pull request, I'll strongly advice against that in your coming projects. You are not meant to close or delete a pull request until it has been approved by a code reviewer.  :+1: :100: 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- LInters setup correctly✔
-  Correct Github flow👋 ✔

 ## Required Changes ♻️

## Optional suggestions
- [OPTIONAL] The PULL REQUEST summary is a brief point that let you tell others about changes you've pushed to a branch in a repository on GitHub. Look at the attached screenshot below, that will be a good pull request summary for this `Hello Microverse` project. :+1: :smile: 

- Screenshot:
Screenshot from 2023-05-31 13-26-05

 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @Frelly0, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**